docs/release_notes.rst

```diff
@@ -38,6 +38,8 @@ repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 Version 1.1.0
 ~~~~~~~~~~~~~
 
+*RTI Connector* 1.1.0 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
```

Comment:
This sentence was missing here (was in JS). 
I do not know if it the correct link or not. @rkorte can you confirm?

Reply:
Yes, that is a valid URL (for 6.1.0 documentation, anyway).